client/volume_inspect.go

```diff
@@ -22,3 +24,24 @@ func (cli *Client) VolumeInspect(ctx context.Context, volumeID string) (types.Vo
 	ensureReaderClosed(resp)
 	return volume, err
 }
+
+// VolumeInspectWithRaw returns the information about a specific volume in the docker host and it's raw representation
+func (cli *Client) VolumeInspectWithRaw(ctx context.Context, volumeID string) (types.Volume, []byte, error) {
```

Comment:
@HackToday I definitely agree with @dnephin. I even think we could do this for the existing ones (in `container_inspect.go` and `image_inspect.go`).